README.md

```diff
@@ -11,10 +11,15 @@ Enhanced History for Chrome
 
 Chrome is an excellent browser, but browsing your history kinda stinks.
 
-Getting rolling
+Manual Installation
 ----------------
 
-1. Clone this repo
-2. Install the repo's directory as an extension
+1. Clone this repo locally or download and unzip
```

Comment:
Can we remove "or download and unzip?" from here? I'd rather not direct people towards that, imo it's better to just clone the repo.

Reply:
Feel free to keep this as we spoke about!